### PR TITLE
🌐 Add German translation for `docs/de/docs/tutorial/first-steps.md`

### PR DESCRIPTION
**Pull request by fhabers21**
add german translation for tutorial/first-steps.md

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 4eb2246124075e45bf7aa9a187df029880ac03b7 at: https://649a2ec1c3bf902028a9e518--fastapi.netlify.app

**Review by creyD — CHANGES_REQUESTED** [review]
Apart from the minor comments I had, I liked the translation.

**Comment by creyD** [comment]
@tiangolo Would it be possible to adjust the docs so that the reader already knows that the article will be in another language than the selected one? Here i.e. there is no translated version yet for the section, so the reader will automatically be redirected to the English one, which is fine, just would be nice to have indication beforehand.
Screenshot 2023-08-31 at 16 19 40

**Comment by fhabers21** [comment]
@creyD Thank you for your valuable feedback. I corrected the points you mentioned.

**Review by creyD — APPROVED** [review]
Thanks for the adjustments, I believe it is good to go now!

**Comment by tiangolo** [comment]
📝 Docs preview for commit 4dca8ea03543eda8a4d9c2efad9c24176b791ec8 at: https://c7d03784.fastapitiangolo.pages.dev

**Comment by nilslindemann** [comment]
Hello @fhabers21, good PR, I approved it.

Could you include the suggestions in the following .md?

[first-steps.md](https://github.com/tiangolo/fastapi/files/12812131/first-steps.md)

These are mostly changes suggested by GeorchW, who approved my other [closed PR](https://github.com/tiangolo/fastapi/pull/10289). Then he could approve this PR here too (ping, @GeorchW), and we can get this merged. :rocket:

**Review by nilslindemann — APPROVED** [review]

File: docs/de/docs/tutorial/first-steps.md [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+
+### Schritt 5: den Inhalt zurückgeben
+
+```Python hl_lines="8"
+{!../../../docs_src/first_steps/tutorial001.py!}
+```
+
+Sie können ein `dict`, eine `list`, einzelne Werte wie `str`, `int`, usw. zurückgeben.
+
+Sie können auch Pydantic Modelle zurückgeben (dazu später mehr).
```
Comment (GeorchW) [inline docs/de/docs/tutorial/first-steps.md:323]:
```suggestion
Sie können auch Pydantic-Modelle zurückgeben (dazu später mehr).
```

**Review by GeorchW — APPROVED** [review]
Nur kleinere Änderungen, eher subjektiver Natur.

**Comment by nilslindemann** [comment]
Notice, there is still a link to https://fastapi.tiangolo.com/async/#in-a-hurry, which will become something like https://fastapi.tiangolo.com/async/#in-eile after that target document is translated. I have marked it as such [here](https://github.com/tiangolo/fastapi/discussions/9203#discussioncomment-7256485).

**Comment by nilslindemann** [comment]
Current state of this document in my work branch, with [above link](https://github.com/tiangolo/fastapi/pull/9530#issuecomment-1758526621) translated:

[first-steps.md](https://github.com/tiangolo/fastapi/files/12919834/first-steps.md)

**Comment by nilslindemann** [comment]
Current state of this document in my work branch:

[first-steps.md](https://github.com/tiangolo/fastapi/files/13064312/first-steps.md)

**Comment by fhabers21** [comment]
@nilslindemann, @GeorchW, big thanks for your valuable feedback. I've included your suggestions.

@tiangolo, we might be good to go.

**Comment by tiangolo** [comment]
📝 Docs preview for commit 9713bcfb3de324713da93b2adb79f830a8d5ed5f at: https://04391ef4.fastapitiangolo.pages.dev

File: docs/de/docs/tutorial/first-steps.md [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+
+    Sie können alle <a href="https://www.starlette.io/" class="external-link" target="_blank">Starlette</a>-Funktionalitäten auch mit `FastAPI` nutzen.
+
+### Schritt 2: Erzeugen einer `FastAPI`-„Instanz“
+
+```Python hl_lines="3"
+{!../../../docs_src/first_steps/tutorial001.py!}
+```
+
+In diesem Beispiel ist die Variable `app` eine "Instanz" der Klasse `FastAPI`.
```
Comment (nilslindemann) [inline docs/de/docs/tutorial/first-steps.md:150]: Hier typografische Anführungszeichen vergessen. („Instanz“)

**Review by nilslindemann — APPROVED** [review]

**Comment by nilslindemann** [comment]
Superklasse! Kann meines Erachtens gemerged werden. Nur noch die eine Stelle, wo die „typografischen“ fehlen.

**Comment by fhabers21** [comment]
@nilslindemann danke für dein erneuten Review! Habe die Stelle ausgebessert.

**Comment by tiangolo** [comment]
📝 Docs preview for commit addad7f40b43671d42632ae480f73601ab0efd98 at: https://d654ffb8.fastapitiangolo.pages.dev

**Comment by nilslindemann** [comment]
Ich hab noch eine Kleinigkeit entdeckt: In Zeile 46 könnte man `JSON-Response` statt `JSON-Antwort` schreiben. Ist aber "Geschmäckle" :-)

**Comment by tiangolo** [comment]
Nice, thank you @fhabers21! 🚀 

And thanks for the reviews @nilslindemann, @GeorchW, @creyD 🙇

**Pull request merged**